internal/engine/eval/jq/jq_test.go

```diff
@@ -292,6 +359,45 @@ func TestValidJQEvals(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "valid single rule evaluates constant array",
+			assertions: []*pb.RuleType_Definition_Eval_JQComparison{
+				{
+					Constant: &structpb.Value{
+						Kind: &structpb.Value_ListValue{
+							ListValue: &structpb.ListValue{
+								Values: []*structpb.Value{
+									{
+										Kind: &structpb.Value_StringValue{
+											StringValue: "a",
+										},
+									},
+									{
+										Kind: &structpb.Value_StringValue{
+											StringValue: "b",
+										},
+									},
+									{
+										Kind: &structpb.Value_StringValue{
+											StringValue: "c",
+										},
+									},
+								},
+							},
+						},
+					},
```

Comment:
I love that we're testing list input. 🎉 

Again, I think the helpers in `structpb` will make this less painful to read and write:

```suggestion
					Constant: &structpb.NewListValue(structpb.ListValue{Values: {
						structpb.NewStringValue("a"),
						structpb.NewStringValue("b"),
						structpb.NewStringValue("c"),
						},
					}}),
```

Unfortunately, `NewList` returns a `*ListValue, error`, so we can't just feed it into `NewListValue` directly.  (We could also do it _outside_ the function, or write an `errorToPanic[T](T, err) T` helper method.)